apps/admin/frontend/src/screens/unconfigured_screen.tsx

```diff
@@ -123,24 +123,23 @@ export function UnconfiguredScreen(): JSX.Element {
       )}
       <P>
         <FileInputButton
-          accept=".json,application/json"
-          onChange={handleVxElectionFile}
+          accept=".zip,application/zip"
+          onChange={handleSetupPackageFile}
+          buttonProps={{ variant: 'primary' }}
         >
-          Select Existing Election Definition File
+          Select Election Package
         </FileInputButton>
```

Comment:
So we're not distinguishing between "Setup Package" and "Election Package"? And they were the same the whole time?